sfja/src/components/StudentProfile/FormsTab.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable max-len */
+import React from 'react';
+import Forms from './Forms';
+import PropTypes from 'prop-types';
+import Chip from '@material-ui/core/Chip';
+
+const chipDivStyle = {
+  display: 'flex',
+  justifyContent: 'center',
+  flexWrap: 'wrap',
+};
+
+const chipStyle = {
+  marginLeft: 5,
+};
+
+// eslint-disable-next-line require-jsdoc
+class FormsTab extends React.Component {
+    static propTypes = {
+      forms: PropTypes.object,
+      studentId: PropTypes.string,
+      history: PropTypes.object,
+    };
+
+    // eslint-disable-next-line require-jsdoc
+    constructor(props) {
+      super(props);
+      // eslint-disable-next-line react/prop-types
+      const {tags} = props;
```

Comment:
@Antranduc tags now also come from the props instead of their own endpoint; this reduces overhead and the information comes down from the `studentProfile` call